Libraries/Components/Picker/PickerAndroid.android.js

```diff
@@ -110,6 +111,12 @@ function PickerAndroid(props: Props): React.Node {
     ],
   );
 
+  // If `no` is specified for `importantForAccessibility`, it will be changed to `no-hide-descendants` because the Picker.Item should not be focused.
```

Comment:
Hey! I just wanted to ask why this is needed? What happens if someone specifies "no" without this code?

Reply:
Thanks. If set to `no`, the Picker will not focus, but it will focus on the TextView inside that.
<img width="401" alt="picker" src="https://user-images.githubusercontent.com/40130327/115146895-89f36b00-a093-11eb-99e1-02a9e640d45d.png">

Reply:
Good catch @grgr-dkrk !

@kacieb , this is due to the fact that while some components like <Button> seem like atomic units, they actually end up rendering a hierarchy of components for example a wrapper <View> with a <Text> inside them.

By allowing those descendants to become focusable, it sort of breaks the model of these being a single component to consider, so I think forcing no-hide-descendants makes sense here.

The other option is to *always* render any descendants of these elements with importantForAccessibility="no", so that they can never be focusable on their own. This would have the same end result, but may potentially cause issues when the descendant content is supposed to automatically get moved up to the focusable ancestor to act as a label (which is what Talkback does with unfocusable text elements by default).